### PR TITLE
fix: No body for 101, 204, 205, or 304 status code

### PR DESCRIPTION
**Pull request by vicb**
Fix a KI when browser cache was enabled by pulling the fix from https://github.com/opennextjs/opennextjs-aws/pull/683

### CODE REVIEW

**Review by petebacondarwin — COMMENTED** [review]
What is a KI?

I feel uncomfortable about landing "fixes" with no tests.
How hard would it be to add tests that prove this bump to the aws dependency fixes the problem?

**Comment by vicb** [comment]
KI = known issue (See the README update).

I tested locally with the blog example, reloading the page would generated the error as it adds a body to a 304. This might be possible to e2e but I'm not sure how e2e behaves regarding to cache.

I'm confident that the issue is fixed.

File: packages/cloudflare/README.md [hunk 1]
```diff
@@ -70,7 +70,6 @@ export default config;
 
 - Next cache is not supported in the experimental branch yet
```
Comment (conico974) [inline packages/cloudflare/README.md:71]: This can be removed ( or changed in the cache PR as well )

**Review by conico974 — APPROVED** [review]
LGTM

File: packages/cloudflare/README.md (hunk 1 above)
Comment (vicb) [reply packages/cloudflare/README.md:71]: ^that should actually go in the #194

**Pull request merged**